### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SimonSimonson123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # playground
 For testing GitHub stuff
+Something new
```
Comment (SimonSimonson123) [inline README.md:3]: Added something new

Comment (xanhex) [reply README.md:3]: Nice work!

Comment (xanhex) [inline README.md:3]: This need to be fixed ASAP!!!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # playground
 For testing GitHub stuff
```
Comment (xanhex) [inline README.md:2]: Some bibas

**Review by xanhex — CHANGES_REQUESTED** [review]
So many bibas in the code! Fix it!

File: README.md (hunk 1 above)
Comment (SimonSimonson123) [reply README.md:3]: Not fixin it!

**Review by xanhex — APPROVED** [review]
Nice bibas

**Pull request merged**